test/OpenTelemetry.Tests/MathHelper.cs

```diff
@@ -0,0 +1,84 @@
+// <copyright file="MathHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#pragma warning disable SA1119 // Statement should not use unnecessary parenthesis
+
+namespace OpenTelemetry.Tests;
+
+internal static class MathHelper
```

Comment:
Move this file to the `Shared` folder in `OpenTelemetry.Tests`?

Reply:
I doubt this will be used elsewhere but I went ahead and moved it.

Reply:
In that case, we could have just kept it in the test file that is using this. Why extract this out into a separate file in the first place?

Reply:
Short answer: Clean code!

Long answer: 

> we could have just kept it in the test file that is using this

The thing is, it isn't used by one test file... it is used by two files! If it was just used in the one I would have left it alone for sure. First I made it `internal static` so one of the test classes could call it inside the other. That is a code smell really indicative of essentially a coupling issue due to separation of concerns principle violation. It also has smelly [suppressions for differing style](https://github.com/open-telemetry/opentelemetry-dotnet/blob/b870ed9b0c965ec89bf6b5aedab87ff3cab8ea68/test/OpenTelemetry.Tests/Metrics/Base2ExponentialBucketHistogram.LowerBoundary.Test.cs#L250-L251) from the code copied from runtime. Too many code smells for me I decided to make it clean[er] 😄